packages/lib-subject-viewers/src/VolumetricViewer/components/Cube.jsx

```diff
@@ -24,11 +24,6 @@ import { useEffect, useLayoutEffect, useRef } from 'react'
 
 // Shim for node.js testing
 const glContext = null
-if (!process.browser) {
-  window.requestAnimationFrame = () => {
-    // needs to be stubbed out for animate() to work
-  }
-}
```

Comment:
`window` is mocked in setup.js for tests instead. Doesn't need to be in the component file.